### DISCUSSION TITLE
Awful red highlighting on perfectly functional markdown text

### DISCUSSION THREAD

**Discussion by Ravencentric** [github-linguist/linguist#6685] [Q&A]
I have no idea why or what's happening. Haven't seen this before but it makes revewing diffs insanely hard
![image](https://github.com/github-linguist/linguist/assets/78981416/9ec9f7d8-9848-46b5-a926-b6c7ae2f15f3)

Example: https://github.com/Snaacky/thewiki/pull/244/files

Repo uses Retype to host markdown pages. Retype extends markdown with custom components https://retype.com/components/

**Comment by lildude** [comment]
If this used to render correctly until late last year, it is quite possibly the known issue with the highlighting engine detailed in https://github.com/github-linguist/linguist/issues/6668

**Reply by Ravencentric** [reply]
Yep, this used to render correctly until last year. Is there anyway for me to disable/fix this?

**Reply by lildude** [reply]
Not really, short of setting the file language to Text using an override (this effectively disables all syntax highlighting) as the issue is with the external syntax highlighting engine.
